rosa_planning/rosa-sts-ocm-role.adoc

```diff
@@ -6,19 +6,40 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 
 toc::[]
 
-{product-title} (ROSA) web UI needs you to have some permissions on your AWS account that create a trust relationship to provide the end-user experience at {cluster-manager-url} and for the `rosa` command line interface (CLI).
+{product-title} (ROSA) web UI requires you to have some permissions on your AWS account that create a trust relationship to provide the end-user experience at {cluster-manager-url} and for the `rosa` command line interface (CLI).
 
-This trust relationship is achieved through the creation and association of two AWS IAM roles:
+This trust relationship is achieved through the creation and association of the `ocm-role` AWS IAM role. This role has a trust policy with the AWS installer that links your Red Hat account to your AWS account. In addition, you also need a `user-role` AWS IAM role for users to access the clusters. This `user-role` AWS IAM role has no permissions.
+
+The AWS IAM roles required to use {cluster-manager} are:
 
 1. `ocm-role`
 2. `user-role`
 
-If you use the `rosa` CLI, the `rosa` tool creates a number of these required permissions for you. This creation is available since your user account authenticates for both Red Hat and AWS. In the {cluster-manager} web UI, you need to create these roles.
+Whether you manage your clusters using the `rosa` CLI or {cluter-manager} web UI, you must create some account-wide roles using the CLI. These account-wide roles authenticate your local shell environment to Red Hat and AWS. You need to create these roles when creating one cluster, but if you are using more clusters, then these roles can be used across your deployment after the initial creation. These required account roles are:
```

Comment:
this didn't get replaced in the preview

Reply:
May be a caching issue. I cleared mine and it displays as:

![image](https://user-images.githubusercontent.com/16167833/197242573-2cc634b2-e683-4b81-b36c-432abf1a43dc.png)